modules/dataquery/js/ui.stats.js

```diff
@@ -1,5 +1,5 @@
 /*global self: false, importScripts: false, jStat: false*/
-importScripts(Loris.BaseURL + "/GetJS.php?Module=dataquery&file=jstat.js");
+importScripts(loris.BaseURL + "/GetJS.php?Module=dataquery&file=jstat.js");
```

Comment:
If you're updating the URL, uou might as well go all the way and change this to the pretty version of loris.BaseURL + "/dataquery/js/jstat.js" while you're at it